diff --git a/node_modules/react-native-geolocation-service/android/build.gradle b/node_modules/react-native-geolocation-service/android/build.gradle
index cf167cf..376ef61 100644
--- a/node_modules/react-native-geolocation-service/android/build.gradle
+++ b/node_modules/react-native-geolocation-service/android/build.gradle
@@ -1,16 +1,17 @@
 apply plugin: 'com.android.library'
 
-def DEFAULT_COMPILE_SDK_VERSION          = 29
-def DEFAULT_BUILD_TOOLS_VERSION          = "29.0.2"
-def DEFAULT_MIN_SDK_VERSION              = 16
-def DEFAULT_TARGET_SDK_VERSION           = 29
-def DEFAULT_GOOGLE_PLAY_SERVICES_VERSION = "18.0.0"
+def DEFAULT_COMPILE_SDK_VERSION          = 35
+def DEFAULT_BUILD_TOOLS_VERSION          = "35.0.0"
+def DEFAULT_MIN_SDK_VERSION              = 24
+def DEFAULT_TARGET_SDK_VERSION           = 35
+def DEFAULT_GOOGLE_PLAY_SERVICES_VERSION = "21.0.1"
 
 def safeExtGet(prop, fallback) {
   rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
 }
 
 android {
+  namespace "com.agontuk.RNFusedLocation"
   compileSdkVersion safeExtGet('compileSdkVersion', DEFAULT_COMPILE_SDK_VERSION)
   buildToolsVersion safeExtGet('buildToolsVersion', DEFAULT_BUILD_TOOLS_VERSION)
 
@@ -19,7 +20,7 @@ android {
     targetSdkVersion safeExtGet('targetSdkVersion', DEFAULT_TARGET_SDK_VERSION)
   }
 
-  lintOptions {
+  lint {
     abortOnError false
   }
 }
diff --git a/node_modules/react-native-geolocation-service/android/src/main/AndroidManifest.xml b/node_modules/react-native-geolocation-service/android/src/main/AndroidManifest.xml
index 22bf78b..9a40236 100644
--- a/node_modules/react-native-geolocation-service/android/src/main/AndroidManifest.xml
+++ b/node_modules/react-native-geolocation-service/android/src/main/AndroidManifest.xml
@@ -1,4 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-  package="com.agontuk.RNFusedLocation">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
 </manifest>
