diff --git a/node_modules/react-native-sensors/android/build.gradle b/node_modules/react-native-sensors/android/build.gradle
index f1526ec..34d0e7b 100644
--- a/node_modules/react-native-sensors/android/build.gradle
+++ b/node_modules/react-native-sensors/android/build.gradle
@@ -23,16 +23,14 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion safeExtGet('compileSdkVersion', 23)
+    namespace "com.sensors"
+    compileSdkVersion safeExtGet('compileSdkVersion', 34)
 
     defaultConfig {
-        minSdkVersion 17
-        targetSdkVersion 29
+        minSdkVersion safeExtGet('minSdkVersion', 24)
+        targetSdkVersion safeExtGet('targetSdkVersion', 34)
         versionCode 1
         versionName "1.0"
-        ndk {
-            abiFilters "armeabi-v7a", "x86"
-        }
     }
     lintOptions {
        warning 'InvalidPackage'
diff --git a/node_modules/react-native-sensors/android/src/main/AndroidManifest.xml b/node_modules/react-native-sensors/android/src/main/AndroidManifest.xml
index 6d0f8b0..0a0938a 100644
--- a/node_modules/react-native-sensors/android/src/main/AndroidManifest.xml
+++ b/node_modules/react-native-sensors/android/src/main/AndroidManifest.xml
@@ -1,5 +1,3 @@
-
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.sensors">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
 </manifest>
